datacube/api/core.py

```diff
@@ -652,6 +637,20 @@ def get_bounds(datasets, crs):
     return geometry.box(left, bottom, right, top, crs=crs)
 
 
+def set_resampling_method(measurements, resampling=None):
```

Comment:
Few concerns with this one:

1. sometimes copies other times returns reference
2. not clear from the name that this is not done by reference, and you have to use the output
3. why can't I set resampling to None exactly?